pkgs/build-support/rust/replace-workspace-values.py

```diff
@@ -96,6 +96,13 @@ def main() -> None:
                 workspace_manifest, crate_manifest["target"][key]
             )
 
+    if (
+        "lints" in crate_manifest
```

Comment:
Shouldn't it be `workspace_manifest` here?

Reply:
No, I'm looking for crates that have `workspace = true` set in the lints block, e.g. see [typst-cli](https://github.com/typst/typst/blob/main/crates/typst-cli/Cargo.toml#L78) here

Reply:
Fair enough, thanks!